www/main/settings/components/SettingViewer.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { useStoreState } from 'easy-peasy';
+
+const SettingViewer = () => {
+  const {
+    gurbaniFontSize,
+    translationFontSize,
+    teekaFontSize,
+    transliterationFontSize,
+    displayNextLine,
+    leftAlign,
+    theme,
+  } = useStoreState(state => state.userSettings);
+
+  const commonStyles = {
+    textAlign: leftAlign ? 'left' : 'center',
+  };
+
+  const gurbaniStyles = {
+    fontSize: `${gurbaniFontSize * 3}px`,
+  };
+
+  const translationStyles = {
+    fontSize: `${translationFontSize * 3}px`,
+  };
+
+  const transliterationStyles = {
+    fontSize: `${transliterationFontSize * 3}px`,
+  };
+
+  const teekaStyles = {
+    fontSize: `${teekaFontSize * 3}px`,
+  };
+
+  const nextLineStyles = {
+    fontSize: `${gurbaniFontSize * 3}px`,
+    opacity: 0.5,
+    display: displayNextLine ? 'block' : 'none',
+  };
+
+  return (
+    <div className={`settings-viewer theme-${theme}`} style={commonStyles}>
+      <h1 className="gurbani gurmukhi" style={gurbaniStyles}>
+        <div>
```

Comment:
![image](https://user-images.githubusercontent.com/2982784/109097685-a77c0800-76d4-11eb-9989-d312447cb939.png)
That doesn't look right.